### PR TITLE
(latam) translation files fillup with Valve translation

### PR DESCRIPTION
**Pull request by Blueberryy**
> [!IMPORTANT]
#1384 Feel free @gaelcoral to suggest changes if some translation that was used in Valve's game is wrong or doesn't fit limits of screen.
When i will be pushing new commit, i will notificate you if you want.

- [x] basemodui file fillup.
- [x] gameui file fillup.
- [x] valve file fillup.
- [x] vgui file fillup.
- [x] serverbrowser file fillup.
- [x] chat file fillup.

> [!NOTE]
> If someone want me to do so for another language - request here in comments below.

### CODE REVIEW

**Comment by Blueberryy** [comment]
Hey @gaelcoral do you want to review filled translation text on this Pull Request or it can be merged and you will do it later if there's any issues with Valve translation?

File: resource/basemodui_latam.txt
Comment (DucDuyGaming) [inline resource/basemodui_latam.txt:2418]:
```suggestion
"L4D_test_mic_tip"		"Probar si tu micrófono funciona correctamente."
```

line 2418 of basemodui contain an extra quote mark. perhaps it is the reason of build error

**Review by DucDuyGaming — CHANGES_REQUESTED** [review]

File: resource/basemodui_latam.txt
Comment (Blueberryy) [reply resource/basemodui_latam.txt]: > line 2418 of basemodui contain an extra quote mark. perhaps it is the reason of build error

Thanks man. But i think it's because this language not enabled in translation sync tool so it might be fixed later after Ben add it to the list.

Comment (DucDuyGaming) [reply resource/basemodui_latam.txt]: alright

Comment (DucDuyGaming) [reply resource/basemodui_latam.txt]: I just wanted to get rid of the build error. Now if by any mean the amount of quote mark is in even number I may not be able to locate it myself, as I used inspect element to locate it instead of some special way. But here are 3 more quote mark error that I found, which I will send it here because I don't want to create another review

in "chat", filter_joinleave contain an extra quote mark
in "gameui", i found that GameUI_MouseFilter also contain an extra one
in "valve", there is also issue at Valve_HDRDEMO_LeftTitle, but instead it's a missing quote mark

I might not be able to help any further if you apply these fixes and it still show error

**Comment by Blueberryy** [comment]
> Hey @gaelcoral do you want to review filled translation text on this Pull Request or it can be merged and you will do it later if there's any issues with Valve translation?

Ay @gaelcoral have you decided?

**Comment by gaelcoral** [comment]
> Hey @gaelcoral do you want to review filled translation text on this Pull Request or it can be merged and you will do it later if there's any issues with Valve translation?

Hi, I have already updated some files of this pull request, except for the basemodui file, which is lengthy and I need more time to review it well. In this case, do I need to make a pull request on your fork in the latam branch, or how do I update those files?

**Comment by gaelcoral** [comment]
> > > Hey @gaelcoral do you want to review filled translation text on this Pull Request or it can be merged and you will do it later if there's any issues with Valve translation?
> > 
> > 
> > Hi, I have already updated some files of this pull request, except for the basemodui file, which is lengthy and I need more time to review it well. In this case, do I need to make a pull request on your fork in the latam branch, or how do I update those files?
> 
> Yeah you need to make fork of my fork to edit it and then create Pull Request on my fork. Or i can ping Ben that this Pull Request is ready to merge and you will edit this like regular fork.

Done

**Pull request merged**
